pages/contacts/_edit.vue

```diff
@@ -435,18 +437,7 @@ function pushToArray(arr, val, newLine, arrVal, arrPrefix) {
   }
 }
 
-</script>
-
-<style>
-.form-control:disabled {
-  background-color: unset;
-  opacity: 0.7;
-  border-color: lightgrey;
-}
-.invalid .typo__label{
-  color: #e55353;;
+function goBack() {
+  this.$router.go(-1);
```

Comment:
Going Back is what the implementation do, the behaviour. The operation is Cancel the edit. Please Rename to `cancel`

```suggestion
function cancel() {
  this.$router.go(-1);
```

Reply:
DONE ! renamed name as suggested